update docs (#26617)
Azure/azure-sdk-for-js: documentation/Quickstart-on-how-to-write-tests.md

The Azure SDK test framework uses the [`test-recorder`](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/README.md) library, which in turn rests upon on a HTTP recording system ([testproxy](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy)) that enables tests dependent on network interaction to be run offline.

> At the moment, notice that this quickstart is based on 3.x.y version of recorder tool (`@azure-tools/test-recorder`).

Reviewer: ```suggestion
Please notice that this quickstart is based on 3.x.y version of recorder tool (`@azure-tools/test-recorder`).
```
